<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLCDNumber" name="lcd_coin_counter">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="to_coins">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TO COINS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="to_bills">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TO BILLS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
